### PR TITLE
stable: add X-Forwarded-User to opsportal and kommander

### PR DESCRIPTION
**Pull request by hectorj2f**
I have the feeling we want to change this too :)

### CODE REVIEW

File: stable/kommander/templates/ingress.yaml [hunk 1]
```diff
@@ -13,7 +13,7 @@ metadata:
     traefik.frontend.rule.type: PathPrefixStrip
     ingress.kubernetes.io/auth-secret: ops-portal-htpasswd
     ingress.kubernetes.io/auth-type: basic
-    ingress.kubernetes.io/auth-header-field: X-WebAuth-User
+    ingress.kubernetes.io/auth-header-field: X-Forwarded-User
```
Comment (jieyu) [inline stable/kommander/templates/ingress.yaml]: I don't think this will work if the `auth-type` is still `basic`.

**Comment by jieyu** [comment]
@brandonc we probably don't need this change, but we need to make sure kommander service expect `X-Forwarded-User` instead of `X-WebAuth-User` from the proxy

File: stable/kommander/templates/ingress.yaml (hunk 1 above)
Comment (jieyu) [reply stable/kommander/templates/ingress.yaml]: actually, scratch that. This is `auth-header-field` not `auth-responses-header`.

File: stable/opsportal/templates/ingress-opsportal.yaml [hunk 2]
```diff
@@ -7,7 +7,7 @@ metadata:
     traefik.frontend.rule.type: PathPrefixStrip
     ingress.kubernetes.io/auth-secret: ops-portal-htpasswd
```
Comment (jieyu) [inline stable/opsportal/templates/ingress-opsportal.yaml]: This has changed. We probably need to remove all those labels and use forward auth instead

Comment (hectorj2f) [reply stable/opsportal/templates/ingress-opsportal.yaml]: done

File: stable/kommander/templates/ingress.yaml [hunk 3]
```diff
@@ -11,9 +11,7 @@ metadata:
   annotations:
     kubernetes.io/ingress.class: traefik
     traefik.frontend.rule.type: PathPrefixStrip
```
Comment (jieyu) [inline stable/kommander/templates/ingress.yaml]: I think this should be configurable by values.yaml. For the konvoy ui page, this annotation should not be PathPrefixStrip.

File: stable/opsportal/templates/ingress-opsportal.yaml [hunk 4]
```diff
@@ -5,9 +5,7 @@ metadata:
   annotations:
     kubernetes.io/ingress.class: traefik
     traefik.frontend.rule.type: PathPrefixStrip
-    ingress.kubernetes.io/auth-secret: ops-portal-htpasswd
-    ingress.kubernetes.io/auth-type: basic
-    ingress.kubernetes.io/auth-header-field: X-WebAuth-User
+    ingress.kubernetes.io/auth-header-field: X-Forwarded-User
```
Comment (jieyu) [inline stable/opsportal/templates/ingress-opsportal.yaml]: Opsportal is going to be replaced by konvoy-ui (based on kommander chart, yes, the same chart with different configurations). So I won't bother do any change in this chart.

**Pull request merged**